webview_panels/src/lib/altimate/main.d.ts

```
@@ -0,0 +1,597 @@
+    className?: string;
+    titleActions?: ReactNode;
+}
+
+declare interface Props_10 {
+    taskLabel: keyof typeof TaskLabels;
+    context?: Record<string, unknown>;
+    onClose: () => void;
+}
+
+declare interface Props_11 {
+}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)